helm/superset/templates/init-job.yaml

```diff
@@ -26,6 +26,10 @@ metadata:
   {{- if .Values.init.jobAnnotations }}
   annotations: {{- toYaml .Values.init.jobAnnotations | nindent 4 }}
   {{- end }}
+  {{- if .Values.istio.enabled }}
```

Comment:
Rather than making this specific to istio, can you leverage the `podLabels` config? Also, istio would likely be injected in the resulting pod, therefore the labels on the `Job` resource would likely be moot

Reply:
I think adding labels in the spec would be the right way to go here...

Reply:
Fair enough.  Added `init.podLabels` and defaulted to `{}` in values.yaml.